.zuul.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- job:
+    name: shellcheck
+    description: Check toolbox with shellcheck
+    nodeset:
+      nodes:
+        - name: ci-node-30
+          label: cloud-fedora-30
+    run: playbooks/shellcheck.yaml
+
+- job:
+    name: test-with-podman-stable
+    description: Test toolbox with stable Podman
+    nodeset:
+      nodes:
+        - name: ci-node-30
+          label: cloud-fedora-30
+    pre-run: playbooks/setup-rpm-env.yaml
+    run: playbooks/test-with-podman-stable.yaml
+
+- job:
+    name: test-with-podman-rawhide
+    description: Test toolbox with latest (rawhide) Podman
+    nodeset:
+      nodes:
+        - name: ci-node-30
+          label: cloud-fedora-30
```

Comment:
We recently added rawhide with this label: `cloud-fedora-rawhide-small` . It seems like this nodeset definition is not consistent with the job name.

Reply:
That's sweet! The Rawhide should indicate that we're testing against the latest Podman. But having a Fedora Rawhide image is surely handy and I'll use it.